packages/main/src/themes/ColorPalette.css

```diff
@@ -2,10 +2,26 @@
 	display: inline-block;
```

Comment:
some mixed tabs abd spaces in ColorPalatte.css, ColorPaletteStaticArea.css files